feat(cli): support CloudFormation simplified resource import (#29087)
aws/aws-cdk: packages/aws-cdk/README.md

For more control over when stack changes are deployed, the CDK can generate a
CloudFormation change set but not execute it.

#### Import existing resources

You can pass the `--import-existing-resources` flag to the `deploy` command:

```console
$ cdk deploy --import-existing-resources
```

> This allows you to automatically import the resources in your template that already
> exist in your AWS account during CloudFormation deployments.
> With this feature, you can reduce the manual effort of import operations and avoid
> deployment failures because of naming conflicts.

Reviewer: ```suggestion
Automatically import resources in your CDK application which represent
unmanaged resources in your account. 
Reduces the manual effort of import operations and avoids
deployment failures due to naming conflicts with unmanaged resources in your account
```

Reviewer: adding some active voice to be more similar to how our other command readme's are written.

> To review which resources are imported or not before actually executing a change set,
> use `--method=prepare-change-set` flag.

Reviewer: ```suggestion
use `--method=prepare-change-set` flag to review which resources are imported or not before deploying
```

You can inspect the change set created by CDK from the management console or other external tools.

```console
$ cdk deploy --import-existing-resources --method=prepare-change-set
```

> To enable this feature only for a specific stack, use `--exclusively` flag.

Reviewer: ```suggestion
Use the `--exclusively` flag to enable this feature for a specific stack
```

```console
$ cdk deploy --import-existing-resources --exclusively StackName
```

> This parameter can only import resources that have custom names in templates. 

Reviewer: ```suggestion
Only resources that have custom names can be imported using `--import-existing-resources`
```

5. When `cdk import` reports success, the resource is managed by CDK. Any subsequent
   changes in the construct configuration will be reflected on the resource.

> NOTE: You can also import existing resources by `--import-existing-resources` option of `cdk deploy` command.

Reviewer: ```suggestion
NOTE: You can also import existing resources by using `--import-existing-resources` flag of the `cdk deploy` command.
```

> This parameter only works for resources that you can set custom physical names,

Reviewer: ```suggestion
Only works for resources that allow custom physical names
```

> such as S3 bucket, DynamoDB table, etc.

Reviewer: ```suggestion
such as S3 bucket, DynamoDB table, etc...
```

Reviewer: Love that you're pushing this change and really appreciate the contribution. I've wanted this functionality for a while so it's great to see it being implemented. Just have some doc changes and it's g2g imo
